venues/MIDL.io/2019/Conference/python/post-submission-stage-2.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/python
+
+import sys, os
+import argparse
+import datetime
+import openreview
+import config
+
+"""
+OPTIONAL SCRIPT ARGUMENTS
+	baseurl -  the URL of the OpenReview server to connect to (live site: https://openreview.net)
+ 	username - the email address of the logging in user
+	password - the user's password
+"""
+
+parser = argparse.ArgumentParser()
+parser.add_argument('--baseurl', help="base URL")
+parser.add_argument('--username')
+parser.add_argument('--password')
+
+args = parser.parse_args()
+
+client = openreview.Client(baseurl=args.baseurl, username=args.username, password=args.password)
+print('connecting to {0}'.format(client.baseurl))
+
+conference = config.get_conference(client)
+
+print('open comment invitations')
+conference.set_authors()
```

Comment:
this looks good to me. the API is looking nice. one comment here is that it's not obvious why the conference needs to set the authors, or what set_authors() does.

(btw, I understand why this is needed, but I'm just making a general comment about the intuitiveness of the API. I don't think it needs to change right now necessarily)

would `create_authors` make sense?

Reply:
May be we don't need to expose this method and create the authors when they are needed:
- When there is at least one invitation that contains a anonynous author group?

I'm not sure.

Reply:
It is odd to have to all this method from outside of the builder.

Reply:
That is a good idea, but I'm not sure if we need to create the author groups in all the conferences. 

For example: for an open conference with public comments we shouldn't create these groups because they are not needed. 

So I want to delay the creation of these groups until we really need them.